Tests/test_pairwise_aligner.py

```diff
@@ -308,7 +308,7 @@ def test_smithwaterman(self):
             str(alignment),
             """\
  AwBw
- |-| 
+ |-|.
 zA-Bz
```

Comment:
This changes the interpretation of the alignment:
` |-| ` means that `AwB` was aligned to `A-B`
` |-|.` means that `AwBw` was aligned to `A-Bz` (with `w` and `z` mismatched to each other).
If you look at the path of the alignment through the trace matrix:
```
>>> alignment.path
((0, 1), (1, 2), (2, 2), (3, 3))
```
you will see that the alignment stops after the third letter. So ` |-| ` is correct here.

Same for the other alignments below I guess.